pkg/testutil/http.go

```diff
@@ -0,0 +1,86 @@
+package testutil
```

Comment:
👍🏻 I'll do a follow-up. I also found in this PR, but not the last one, the `threadsafe.Buffer` type, which can replace the `testutil.MockStdout`, so that can also be consolidated

Reply:
Awesome! Thanks. When I saw `MockStdout` I was like... I'm sure I've fixed this before 🤔 😅